web/src/main/java/org/akaza/openclinica/control/core/SecureController.java

```diff
@@ -704,28 +704,24 @@ else if (currentRole.getId() > 0
             // Set if enrollment is capped. Used by navBar.jsp to hide "Add Participant" link in the menu
             processRequest();
         } catch (InconsistentStateException ise) {
-            ise.printStackTrace();
-            logger.warn("InconsistentStateException: org.akaza.openclinica.control.SecureController: " + ise.getMessage());
-
+            logger.warn("InconsistentStateException: org.akaza.openclinica.control.SecureController: {}" , ise.getMessage(), ise);
```

Comment:
you don't need to print the message separately. Passing exception in the argument would print the message as well

Reply:
I have changed according to your comments